main.go

```diff
@@ -26,29 +26,33 @@ func main() {
 		c.File("static/index.html")
 	})
 
+	router.GET("/task/", func(c *gin.Context){
+		err, allTasks := service.GetAllTasks()
+		if err != nil {
+			c.JSON(http.StatusBadRequest, "Could not get all tasks");
```

Comment:
This is not bad reques, but server error?

Reply:
yes, makes sense